terraform/cloudwatch.tf

```diff
@@ -2,3 +2,31 @@ resource aws_cloudwatch_log_group vphone {
   name              = "vphone"
   retention_in_days = 14
 }
+
+module "notify_slack" {
```

Comment:
Follow conventions for the rest of the project and remove double quotes from resource names.